tests/phpunit/integration/Core/Util/Feature_FlagsTest.php

```diff
@@ -70,23 +70,23 @@ public function data_is_feature_enabled() {
 		return array(
 			'no feature given'                             => array(
 				array(
-					'test_feature' => array( 'enabled' => 'production' ),
+					'test_feature' => 'production',
```

Comment:
These tests can be simplified, again since the config will always be a flat map of `[ key ]: [ (string | array) environments ]`